deployment/common/nginx.conf

```diff
@@ -22,7 +22,7 @@ server {
      # we don't want nginx trying to do something clever with
      # redirects, we set the Host: header above already.
      proxy_redirect off;
-     proxy_pass http://merginmaps-web:8080;
+     proxy_pass http://web:8080;
```

Comment:
where we dropped this?

Reply:
Only service name is reachable.  `container_name` is no enough, we would also need to set `hostname` on compose level. This might be something related with new compose or docker engine version has it I remember it worked in the past, just with `container_name` set.

Reply:
it would be good to know if this is due to compose upgrade and if the change is backward compatible else we should fix compose version

Reply:
@varmar05 Sorry, I now rememeber the change. This is to include the `nginx.conf` under the `common` folder, so the only common thing between community and enterprise deployments is the service name. For enterprise, we add an extra `*-enterprise` for containers names.